scripts/validate_test_schemas.py

```diff
@@ -0,0 +1,137 @@
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import time
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+
+def check_connection():
+    checks = 4
+    while checks > 0:
+        response = subprocess.run(
+            [
+                "curl",
+                "-so",
+                "/dev/null",
+                "-w",
+                "%{http_code}",
+                "http://localhost:5002/status",
+            ],
+            capture_output=True,
+            text=True,
+            check=False,
+        ).stdout.strip()
+
+        if response != "200":
+            logging.error("\033[31m---Error: Schema Validator Not Reachable---\033[0m")
+            logging.error("\033[31mHTTP Status: %s\033[0m", response)
+            if checks != 1:
+                logging.info("Retrying...\n")
+                time.sleep(5)
+            else:
+                logging.info("Exiting...\n")
+                sys.exit(1)
+            checks -= 1
+        else:
+            checks = 0
+
+
+def get_schemas() -> list[str]:
+    if len(sys.argv) == 1 or sys.argv[1] == "--local":
+        file_path = "./schemas/test/en"
+        schemas = [
+            os.path.join(file_path, f)
+            for f in os.listdir(file_path)
+            if f.endswith(".json")
+        ]
+        logging.info("--- Testing Schemas in %s ---", file_path)
+    else:
+        file_path = sys.argv[1]
+        schemas = [sys.argv[1]]
+        logging.info("--- Testing %s Schema ---", file_path)
```

Comment:
Minor, can just be schema here rather than calling arg twice:
```suggestion
        schema = sys.argv[1]
        schemas = [schema]
        logging.info("--- Testing %s Schema ---", schema)
```